### PR TITLE
Added log results as part of pipeline + enhance directly from test loader

### PR DESCRIPTION
**Pull request by mosem**
In order to add log results as part of pipeline, it was easier to read clean files from directory.
For that, I had the enhance function accept the test loader directly with paths and save clean files as well as noisy and enhanced files in output directory.

### CODE REVIEW

File: denoiser/enhance.py [hunk 1]
```diff
@@ -7,39 +7,32 @@
...
 
-def get_estimate_and_trim(model, noisy, target_length):
+def get_estimate(model, noisy):
     torch.set_num_threads(1)
     with torch.no_grad():
         estimate = model(noisy)
-    if estimate.shape[-1] > target_length:
-        # logger.info(f'trimming {estimate.shape[-1] - target_length}')
-        estimate = estimate[..., :target_length]
     return estimate
```
Comment (Or-Tal) [inline denoiser/enhance.py:26]: We need get estimate and trim function as it was previously designed as this may not support full audio-length files.
In addition, we need to add padding support estimate.shape[-1] < target_length

**Comment by mosem** [comment]
Look at updated trim mechanism.
